src/picongpu/include/simulation_defines/unitless/physicalConstants.unitless

```diff
@@ -30,38 +30,14 @@ namespace picongpu
     /** Unit of length */
     BOOST_CONSTEXPR_OR_CONST float_64 UNIT_LENGTH = UNIT_TIME*UNIT_SPEED;
 
-#if(SIMDIM==DIM3)
-    namespace SI
-    {
-        /** density normed to dimension of the simulation*/
-        BOOST_CONSTEXPR_OR_CONST float_64  GAS_DENSITY_NORMED= SI::GAS_DENSITY_SI;
```

Comment:
`GAS_DENSITY_NORMED` is needed by [gasConfig.unitless](https://github.com/ComputationalRadiationPhysics/picongpu/pull/1601/files#diff-236b4e3b436ad7ebb3cf9a6d32d7fefeR30)